round4/team-13.md

```diff
@@ -0,0 +1,57 @@
+# Chrome Extension Idea: TypeMaster (tentative)
```

Comment:
I think this would be a very helpful tool that I would personally use. There's so many instances where I will paste something into Google Docs to check the word count. However, another reason I will first type something into Google Docs is to make sure I don't lose what I write in case the website glitches. Perhaps you could consider adding a feature to also save users' work on their respective websites.

I'm also curious in what visual indicators you will be using to track the word count. Perhaps you could use a progress bar or different colors surrounding a text field to indicate progress. 

As for the analytics, I think that could be incredibly helpful in allowing users to check the trends of their productivity. I would consider looking into using [chart.js](https://www.chartjs.org/) to graph word count over time.

Reply:
Yeah we were thinking a progress bar and some sort of celebratory animation when a "word milestone" is reached. I also really like the idea of saving the users work and the analytics side!

Reply:
Saving a user's work + including analytics sounds awesome! Will definetely check out chart.js!